scripts/lib/libvirt/libvirt_setup.py

```diff
@@ -56,15 +56,24 @@ def get_machine_arch():
     return os.uname()[4]
 
 
-def get_os_loader():
-    path = '/usr/share/qemu/aavmf-aarch64-code.bin'
+def get_os_loader(firmware_type=""):
     if 'aarch64' in get_machine_arch():
-        return """
-  <loader readonly='yes' type='pflash'>%s</loader>
-""" % path
+        path = '/usr/share/qemu/aavmf-aarch64-code.bin'
+        return """<loader readonly='yes' type='pflash'>%s</loader>""" % path
```

Comment:
Do not use three quotes for strings that do not contains new lines. Because there are inside single quotes, use only " to wrap this.